syft/serde/protobuf/native_serde.py

```diff
@@ -35,6 +36,28 @@ def _unbufferize_none(worker: AbstractWorker, obj: "Empty") -> "type(None)":
     """
     return None
 
+def _bufferize_type(worker: AbstractWorker, obj) -> ClassTypePB:
+    proto_type = ClassTypePB()
+
+    if isinstance(obj, type):
+        serialized_type = syft.serde.msgpack.serde._simplify(worker, obj)
```

Comment:
The Protobuf serde shouldn't depend on the msgpack serde. (We plan to replace msgpack with Protobuf, so the msgpack serde will eventually go away.)